powerline/selectors/tmux.py

```diff
@@ -0,0 +1,32 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+from powerline.bindings.tmux import get_tmux_output
+import subprocess
+import re
+
+_cached_clients = {}
+# This Regex parses pstree -s <Process_ID> information
+# It will match, if any process name is sshd
+SSHD_REGEX = re.compile(r"(?:^|\W)sshd(?:^|\W)")
+
+
+def tmux_via_ssh(pl, segment_info, mode, caching=True, default=False):
+	'''Returns True if tmux client is connected via SSH
+	'''
+	pid = get_tmux_output(pl, "display", "-p", "#{client_pid}")
```

Comment:
Is there a better way of doing it?
For me this worked okay-ish (it got the `display -p #{client_pid}` output for the current active tmux window so that it would not show excluded segments when I was connected via SSH to the machine and working actively on it), also when using the powerline-daemon

Reply:
You need another tmux instance *not* run from ssh for that to not be OK. This does not mean that variant is acceptable, you would need to refactor relevant code.